File: management/src/main/java/io/hexaforce/management/controller/AcceptanceReportController.java
package io.hexaforce.management.controller;

import java.util.Arrays;
import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.http.HttpStatus;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.ResponseBody;
import org.springframework.web.bind.annotation.RestController;

import io.hexaforce.dievas.commons.DievasServerMesseage;
import io.hexaforce.dievas.service.AcceptanceReportService;
import lombok.extern.slf4j.Slf4j;

/**
 * AcceptanceReport router function.
 * 
 * @version 1.0.0.BUILD-SNAPSHOT
 * @author T.Tantaka 
 */

@Slf4j
@RestController
@RequestMapping("/api")
public class AcceptanceReportController {
	
	@Autowired
	private AcceptanceReportService acceptanceReportService;
	
	@ResponseBody
	@GetMapping(value = "/acceptance-report")
	public ResponseEntity<List<DievasServerMesseage>> init() {
		log.info("Controller(java): init()");
		String x = acceptanceReportService.init();
		return new ResponseEntity<List<DievasServerMesseage>>(Arrays.asList(new DievasServerMesseage(x)),HttpStatus.OK);
	}
	
}

File: webapp/src/main/resources/static/controller/AcceptanceReportController.js
'use strict';
/*******************************************************************************
 * AcceptanceReportController
 ******************************************************************************/
app.controller('AcceptanceReportController', ['AcceptanceReportService', '$scope', function (AcceptanceReportService, $scope) {
  
  $scope.init = function () {
    console.log('AcceptanceReportController.js: init()');
    AcceptanceReportService.init();
    $scope.serviceName = AcceptanceReportService.getInitResult()[0].message;
  }
  
}]);